src/redux/Books/book.js

```diff
@@ -2,17 +2,49 @@ import { createSlice } from '@reduxjs/toolkit';
 
 const bookSlice = createSlice({
   name: 'books',
-  initialState: [],
+  initialState: [
+    {
+      item_id: 'item1',
+      title: 'The Great Gatsby',
+      author: 'John Smith',
+      category: 'Fiction',
+    },
+    {
+      item_id: 'item2',
+      title: 'Anna Karenina',
+      author: 'Leo Tolstoy',
+      category: 'Fiction',
+    },
+    {
+      item_id: 'item3',
+      title: 'The Selfish Gene',
+      author: 'Richard Dawkins',
+      category: 'Nonfiction',
+    },
+  ],
 
   reducers: {
     addBook: (state, action) => {
       const newBook = {
-        id: Date(),
+        id: action.payload.item_id,
         title: action.payload.title,
+        author: action.payload.author,
+        category: action.payload.category,
       };
       state.push(newBook);
     },
-    removeBook: (state, action) => state.filter((book) => book.id !== action.payload.id),
+    removeBook: (state, action) => {
+      const itemId = action.payload.item_id;
+      const index = state.findIndex((book) => book.id === itemId);
```

Comment:
- [ ] Great job so far @Daniel-Kigozi There are just a few changes you need to make and you'll be on your way to the next milestone 😉 
- [ ] I noticed you fixed the first issue you fixed the first issue described by the first reviewer. Hoowever i noticed that even though all the items aren't removed when one is removed, the right item is not removed when clicked on. From the video below you can see that when I click on the last item the first item is removed so kindly fix this as well so that when a book is removed, that book will be removed and not a different book 👍 
- Video Reference 

https://github.com/Daniel-Kigozi/react-book2/assets/102699958/46489f75-b7e1-4d0e-8e94-bbdab8b5a90e